Server/Controllers/BookController.cs

```diff
@@ -27,15 +26,15 @@ public BookController(ILogger<BookController> logger)
         }
 
         [HttpPost]
-        public ActionResult<Book> Create(Book book)
-        {
-            Publisher publisher = dataStorage.GetPublisher(book.BookPublisherId);
-            book.BookPublisher.PublisherId = publisher.PublisherId;
-            book.BookPublisher.PublisherName = publisher.PublisherName;
-            books.Add(book);
-            dataStorage.AddBookToList(book.BookPublisherId, book);
-            return book;
-        }
+        //public ActionResult<Book> Create(Book book)
```

Comment:
For a study, committing commented out code is fine (for further referencing). However, this practice should be avoided in commercial products. Reasoning - version control itself has the history available, there is no need to clutter the source code file with dead code.